File: Planta.java
package Trabajo;

public class Planta {
    
    public String nombre;

    public String forma;

    public String textura;

    public Planta(String nom, String form, String tex)
    {
        establecerNombre(nom);
        establecerForma(form);
        establecerTextura(tex);
    }

    //nombre
    public String obtenerNombre()
    {
        return nombre;
    }

    public void establecerNombre(String nom)
    {
        nombre = nom;
    }

    //genero
    public String obtenerForma()
    {
        return forma;
    }

    public void establecerForma(String form)
    {
        forma=form;
    }

    //color
    public String obtenerTextura()
    {
        return textura;
    }
    public void establecerTextura(String tex)
    {
        textura = tex;
    }
}

File: Planta.php
<?php

namespace App\Model;

class Planta
{
    public $forma = "";
    public $textura = "";
    public $fruto = "";

    //forma
    public function __construct($form)
    {
       $this->establecerForma();
    }

    public function obtenerForma()
    {
        return $this->forma;
    }

    public function establecerColor($form)
    {
        $this->forma = $form;
    }

    //textura
    public function __construct($tex)
    {
       $this->establecerTextura();
    }

    public function obtenerTextura()
    {
        return $this->textura;
    }

    public function establecerEmail($tex)
    {
        $this->textura = $tex;
    }

    //fruto
    public function __construct(fru)
    {
       $this->establecerFruta();
    }

    public function obtenerFruta()
    {
        return $this->fruta;
    }

    public function establecerMarca($fru)
    {
        $this->fruta = $fru;
    }
}